tensorflow/core/kernels/gpu_prim_helpers.h

```diff
@@ -57,6 +58,36 @@ Status GpuRadixSort(OpKernelContext* context, int size, const Tkey* keys_in,
                     const Tindex* indices_in,  // Optional
                     Tindex* indices_out, int num_bits = sizeof(Tkey) * 8) {
   if (size == 0) return Status::OK();
+  if (num_bits == 0) {
```

Comment:
Would setting num_bits to 1 also work instead of having this logic? It would simplify the logic.

Reply:
I was going to do that but realized that `num_bits=0` does not necessarily mean that all keys are zero, and if they're not then forcing `num_bits=1` gives a different result (e.g., the `gpu_prim_helpers_test.cu.cc` test will fail).